src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/support/LicenceTypeOverrideServiceTest.kt

```diff
@@ -315,7 +315,7 @@ class LicenceTypeOverrideServiceTest {
       argumentCaptor<AuditEvent>().apply {
         verify(auditEventRepository, times(1)).saveAndFlush(capture())
         assertThat(firstValue.detail).isEqualTo("ID 1 type PSS status IN_PROGRESS version 1.1")
-        assertThat(firstValue.username).isEqualTo("smills")
+        assertThat(firstValue.username).isEqualTo("tcom")
         assertThat(firstValue.summary).isEqualTo("Licence type overridden for John Smith: from AP_PSS to PSS: Test change to PSS")
```

Comment:
Do we care about `John Smith`? 😅

Reply:
Same as above, John Smith being an idiom, I think it's fine to leave